target/linux/mediatek/filogic/base-files/lib/preinit/80_switch_to_oem_eeprom.sh

```diff
@@ -0,0 +1,21 @@
+. /lib/functions/system.sh
+
+preinit_switch_to_oem_eeprom() {
+	board=$(board_name)
+
+	case "$board" in
+	mercusys,mr90x-v1)
+		eeprom="/lib/firmware/mediatek/mt7986_eeprom_mt7975_dual.bin"
+		eeprom_oem="/tmp/tp_data/MT7986_EEPROM.bin"
+		if ! [[ -L $eeprom ]] && [ -e $eeprom_oem ];
```

Comment:
Can't this be placed in 11-mt76-caldata? Like this:
Apply the patch first (https://github.com/aiamadeus/openwrt/commit/6bb1f582c3a01dcc9d2e60c18107ff5803fba37c).
```
--- a/target/linux/mediatek/filogic/base-files/etc/hotplug.d/firmware/11-mt76-caldata
+++ b/target/linux/mediatek/filogic/base-files/etc/hotplug.d/firmware/11-mt76-caldata
@@ -7,6 +7,13 @@
 board=$(board_name)
 
 case "$FIRMWARE" in
+"mediatek/mt7986_eeprom_mt7975_dual.bin"
+       mercusys,mr90x-v1)
+               ln -sf /tmp/tp_data/MT7986_EEPROM.bin \
+                       /lib/firmware/$FIRMWARE
+               ;;
+       esac
+       ;;
 "mediatek/mt7986_eeprom_mt7976_dbdc.bin")
        case "$board" in
        asus,tuf-ax4200)
```

Reply:
I tried before your review without success and retried again with your patch.
If `mediatek/mt7986_eeprom_mt7975_dual.bin` doesn't exist:
```
Sun May 14 08:23:47 2023 kern.warn kernel: [   12.062432] mt7986-wmac 18000000.wmac: eeprom load fail, use default bin
Sun May 14 08:23:47 2023 kern.warn kernel: [   12.070286] mt7986-wmac 18000000.wmac: Direct firmware load for mediatek/mt7986_eeprom_mt7975_dual.bin failed with error -2
Sun May 14 08:23:47 2023 kern.warn kernel: [   12.081397] mt7986-wmac 18000000.wmac: Falling back to sysfs fallback for: mediatek/mt7986_eeprom_mt7975_dual.bin
Sun May 14 08:23:47 2023 kern.warn kernel: [   12.212195] mt7986-wmac: probe of 18000000.wmac failed with error -12
```
If `mediatek/mt7986_eeprom_mt7975_dual.bin` exist then driver use it.

So I'll leave the current version for now. My sorry.